main.py

```diff
@@ -1,10 +1,56 @@
 import init_django_orm  # noqa: F401
 
+import json
+
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as players:
+        all_players = json.load(players)
+
+    for player_name, player_info in all_players.items():
+        race = player_info["race"]
+        skills = race["skills"]
+        guild = player_info["guild"]
+
+        try:
```

Comment:
I think this `try`-`except` block is difficult to read. Could you rewrite it using the `if`-`else` block?